Tasks/Go/task.json

```diff
@@ -0,0 +1,79 @@
+{
+    "id": "34B37FDD-BBF7-4EF1-B37C-9652CA7BB355",
+    "name": "Go",
+    "friendlyName": "Go",
+    "description": "Get, build, test a go application, or run a custom go command.",
+    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?linkid=867582)",
+    "category": "Package",
+    "author": "Microsoft Corporation",
+    "preview": true,
+    "version": {
+        "Major": 0,
+        "Minor": 1,
+        "Patch": 0
+    },
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
```

Comment:
Lets keep it consistent with .Net Core and have it only for agent. Can you confirm from Alok